### PR TITLE
Designed PR template for repository. Issue #52

### PR DESCRIPTION
**Pull request by tanmay-S12**
Designed PR template for the repository as per the requirements. **Issue**  #52 . Hope you like it, if need some changes, pls let me know. (Updated)

### CODE REVIEW

File: .docs/PR_template.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!--If you need help with you PR or if you want to to discuss something with us, join the **WalkingPal** [Discord community](https://discord.gg/Et3aQTsW6a)!-->
+* ### If you need help with you PR or if you want to to discuss something with us, join the **WalkingPal** [Discord community](https://discord.gg/Et3aQTsW6a)!
+   ---------------------------------------------------
+
+
+   ### Checklist : 
+   <!--Put 'x' in the boxes that apply.!-->
```
Comment (ashuvssut) [inline .docs/PR_template.md:7]: These 2 lines are not needed

File: .docs/PR_template.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+  - [ ] I have read and I am following [the guidelines for creating a new Pull Request.](CONTRIBUTING.md)
+   ---------------------------------------------------
+
+ * ### Link this PR to your issues by writing: fixes #issue_id OR closes #issue_id
```
Comment (ashuvssut) [inline .docs/PR_template.md:12]: this line is not needed

File: .docs/PR_template.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<!--If you need help with you PR or if you want to to discuss something with us, join the **WalkingPal** [Discord community](https://discord.gg/Et3aQTsW6a)!-->
+* ### If you need help with you PR or if you want to to discuss something with us, join the **WalkingPal** [Discord community](https://discord.gg/Et3aQTsW6a)!
```
Comment (ashuvssut) [inline .docs/PR_template.md:2]: This line is not needed

**Review by ashuvssut — CHANGES_REQUESTED** [review]

**Comment by ashuvssut** [comment]
`.docs` is not where PR templates are kept. you make use of `.github` folder and put `.yml` / `.md` files and stuff.

image
refer https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository

**Comment by tanmay-S12** [comment]
@ashuvssut I have committed the changes. Please check it now.

**Comment by ashuvssut** [comment]
See the screenshot from the docs I showed you. follow that

**Comment by tanmay-S12** [comment]
Ohkay, I will make in into . GitHub/ pr template folder

**Comment by tanmay-S12** [comment]
@ashuvssut Please refer to PR #93. I have made the requested changes to it.

**Comment by ashuvssut** [comment]
closing this PR. changes merged via #93 .

**Pull request closed without merging**